src/main/java/com/lmax/tool/disruptor/reflect/ReflectiveRingBufferInvocationHandler.java

```diff
@@ -51,6 +53,11 @@
     @Override
     public Object invoke(final Object proxy, final Method method, final Object[] args) throws Throwable
     {
+        if (TO_STRING_METHOD_NAME.equals(method.getName()))
```

Comment:
To be completely accurate, I think this check should also check the number of parameters to the call is zero.  An overload of Foo.toString(long l) may need to be proxied.